engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -785,6 +812,15 @@ void paladin_t::trigger_t28_4p_prot( action_state_t* s )
   cooldowns.t28_4p_prot_icd->start();
 }
 
+void paladin_t::trigger_tyrs_enforcer( action_state_t* s )
+{
+  // escape if we don't have Tyrs Enforcer
+  if ( !talents.tyrs_enforcer->ok() )
+    return;
+  active.tyrs_enforcer_damage->set_target( s->action->player );
```

Comment:
This would cause it to trigger on the player, but I think you want this to trigger on the mob.

set_target ( s -> target );